snapcraft/plugins/rust.py

```diff
@@ -152,6 +152,13 @@ def _fetch_rust(self):
                   '--disable-sudo', '--save'] + options)
 
     def _fetch_deps(self):
+        
+        source_subdir = getattr(self.options, 'source_subdir', None)
```

Comment:
@sergiusens , @elopio showed me how to make it work so now source_subdir is part of the options class and this suggestion works! I think it makes the code even better, because self.options.source_subdir can be used in the future if there's any other use for it. thank you!